.rubocop.yml

```diff
@@ -0,0 +1,27 @@
+AllCops:
+  Exclude:     
+    - 'db/**/*'     
+    - 'config/**/*'     
+    - 'script/**/*'
+    - 'test/**/*.rb'
+    - '**/*.rake'
+    - 'lib/tasks/auto_annotate_models.rake'
+    - 'bin/**/*'
+    - 'Rakefile'
+    - 'vendor/**/*'
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+
+ClassAndModuleChildren:
+  EnforcedStyle: nested
+
+Style/SymbolArray:
+  Enabled: true
+  EnforcedStyle: brackets
```

Comment:
newline

Reply:
both Enabled & EnforcedStyle relates to the Style/SymbolArray